animations/lib/main.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'src/basics/animation_controller_demo.dart';
+import 'src/misc/expand_card.dart';
+
+void main() => runApp(AnimationsSamples());
+
+class Demo {
+  final String name;
+  final String route;
+  final WidgetBuilder builder;
+
+  Demo(this.name, this.route, this.builder);
+}
+
+List<Demo> basicDemos = [
```

Comment:
nit: `final` is more common in the repo than an explicit type for simple things like these lists:

`final basicDemos = [...]`

For the maps, the function that controls how their types are ultimately inferred is complex-looking enough that an explicit type can help with readability (though they could be final as well, I believe).